.circleci/config.yml

```diff
@@ -91,24 +91,19 @@ jobs:
           paths:
             - docker-images
   migrate_contracts:
-    executor: docker-thesis-buildpack
+    executor: docker-node
     steps:
       - setup_remote_docker:
           docker_layer_caching: true
       - checkout
-      - run:
-          name: Set Gcloud Auth
-          command: |
-            echo $GCLOUD_SERVICE_KEY_JUMPHOST_SSH > ~/gcloud-service-key.json
-            gcloud auth activate-service-account --key-file ~/gcloud-service-key.json
       - run:
           name: Migrate Contracts
           command: |
-            export BUILD_TAG=$CIRCLE_SHA1
-            export TRUFFLE_NETWORK=$TRUFFLE_NETWORK
-            export CONTRACT_OWNER_ETH_ACCOUNT_PRIVATE_KEY=$CONTRACT_OWNER_ETH_ACCOUNT_PRIVATE_KEY
             mkdir -p /tmp/keep-client/contracts
-            solidity/scripts/circleci-migrate-contracts.sh
+            cd ./solidity
+            npm ci
+            ./node_modules/.bin/truffle migrate --reset --network $TRUFFLE_NETWORK
```

Comment:
Notably, we lost tenderly here.

Reply:
😬 

What I think we need to do is to add:
```
tenderly login --authentication-method token --token $TENDERLY_TOKEN
tenderly push --networks $ETH_NETWORK_ID --tag keep-core \
    --tag $GOOGLE_PROJECT_NAME --tag $CIRCLE_SHA1 || echo "tendery push failed :("
```
here and remove `circleci-migrate-contract.sh` altogether since we no longer use it anywhere.

Reply:
Please see https://github.com/keep-network/keep-core/pull/1972/commits/9166ac59605c274e5bfbc189be8eb45a8a0d3c8c and https://github.com/keep-network/keep-core/pull/1972/commits/5049f18309a2e2b005a1b72768fcd799cd8cf3c3.

Reply:
^ Those two commits do no longer reflect the whole solution. Tenderly is back and it's configured in contracts migration shell script.